### DISCUSSION TITLE
Fairy ring Favorites?

### DISCUSSION THREAD

**Discussion by illillusion** [runelite/runelite#15838] [Q&A] [answered]
I am just a peon, just had an idea for what i think is a simple plug in?
We have world favorites, just 2 and thats fine... was wondering if it would be "easy" to code a fairy ring favorite tab.

im just afk fishing between DKP and DJR to bank.
im sure it could be helpful for ironmen, long grinds, ect.

i only took the time to throw this peon idea on here because i think alot of people would down load it if it exsisted.
maybe it does? i looked though fairy ring discussions vaguely.
 hell, thanks for reading my suggestion.

**Comment by Enriath** [comment, marked as answer]
The game already allows you to select up to 4 known fairy rings to display at the top of the travel log.
